src/components/TimeLine.vue

```diff
@@ -39,10 +40,29 @@ export default defineComponent({
         x: event.clientX - controlsWidth.value,
         y: event.clientY + 20,
       };
-
-      tooltipMessage.value = `Slice: ${key}
+      if (isDate.value) {
+        tooltipMessage.value = `Slice: ${key}
+      Time: ${new Date(slice[0]).toLocaleString(undefined, {
+    year: 'numeric',
+    month: 'short',
+    day: 'numeric',
+    hour: 'numeric',
+    minute: 'numeric',
+    timeZoneName: 'short',
```

Comment:
You could extract this object into a config variable instead of declaring it twice in the string interpolation

Reply:
Actually, you use these `.toLocaleString` calls in several places. Maybe you could make them into helper functions in a utils file that you call out to. Instead of just declaring them inline. It would make it easier to reuse them that way

Reply:
yeah that's a good suggestion!